packages/opencensus-exporter-zpages/test/test-zpages.ts

```diff
@@ -0,0 +1,143 @@
+    });
+    it('Should create predefined span names in the zpages', () => {
+      for (const name of options.spanNames) {
+        /** Array within all same name spans */
+        const spans = zpages.getTraces(name);
+        /** Check if the first position span has the same name */
+        assert.equal(spans[0].name, name);
+      }
+    });
+  });
+
+  /** Should start a new span and get it with zpages */
+  describe('starting a new span', () => {
+    /** Creating here because tracing is a singleton */
+    const tracing = require('@opencensus/nodejs');
+    const zpages = new ZpagesExporter(options);
+
+    tracing.start(defaultConfig);
+    tracing.registerExporter(zpages);
+
+    tracing.tracer.startRootSpan({name: 'rootSpanTest'}, (rootSpan) => {
+      const span = tracing.tracer.startChildSpan('spanNameTest', 'spanType');
+      span.end();
+      rootSpan.end();
+    });
+    it('Should create span in the zpages', () => {
+      /** Array within all same name spans */
+      let spans = zpages.getTraces('rootSpanTest');
+      /** Check if the first position span has the same name */
+      assert.equal(spans[0].name, 'rootSpanTest');
+
+      /** Array within all same name spans */
+      spans = zpages.getTraces('spanNameTest');
+      /** Check if the first position span has the same name */
+      assert.equal(spans[0].name, 'spanNameTest');
+    });
+  });
+
+  describe('catching a running span', () => {
+    /** Creating here because tracing is a singleton */
+    const tracing = require('@opencensus/nodejs');
+    const zpages = new ZpagesExporter(options);
+
+    tracing.start(defaultConfig);
+    tracing.registerExporter(zpages);
+
+    tracing.tracer.startRootSpan({name: 'runningSpanTest'}, () => {});
+
+    it('Should get a running span in the zpages', () => {
+      /** Array within all same name spans */
+      const spans = zpages.getTraces('runningSpanTest');
+      /** Check if the first position span has the same name */
+      assert.equal(spans[0].name, 'runningSpanTest');
+      assert.ok(spans[0].started);
+      assert.ok(!spans[0].ended);
+    });
+  });
+
+  describe('running Zpages Server', () => {
+    let zpages;
+
+    /** Starting the server */
+    before((done) => {
+      zpages = new ZpagesExporter(options);
+      zpages.startServer(done);
+    });
+
+    it('Should access tracez page', (done) => {
+      http.get(zpagesServerUrl + '/tracez', (res) => {
+            assert.equal(res.statusCode, 200);
+            done();
+          }).on('error', (e) => {
+        /** Force wrong assert and return error message */
+        assert.equal(e.message, 200);
+        done();
+      });
+    });
+
+    it('Should access tracez page', (done) => {
+      http.get(zpagesServerUrl + '/traceconfigz', (res) => {
+            assert.equal(res.statusCode, 200);
+            done();
+          }).on('error', (e) => {
+        /** Force wrong assert and return error message */
+        assert.equal(e.message, 200);
```

Comment:
You can just do `done(e)` or `.on('error', done)`.

Reply:
Right. Changed all occurrences.